tests/components/cambridge_audio/test_media_player.py

```diff
@@ -128,6 +129,29 @@ async def test_entity_supported_features(
     )
 
 
+async def test_entity_supported_features_with_controlBus(
```

Comment:
Please fix the name on this test.

```suggestion
async def test_entity_supported_features_with_control_bus(
```

Reply:
Done